crate2nix.nix

```diff
@@ -0,0 +1,66 @@
+{ callPackage, stdenv, lib, darwin
+, defaultCrateOverrides, cargo, nix, nix-prefetch-git, makeWrapper, symlinkJoin
+, release ? true}:
+let
+  cargoNix = callPackage ./crate2nix/Cargo.nix { inherit release; };
+  withoutTemplates = name: type:
+  let
+    baseName = builtins.baseNameOf (builtins.toString name);
+  in
+  !(baseName == "templates" && type == "directory");
+  crate2nix = cargoNix.rootCrate.build.override {
+    testCrateFlags = [
+      "--skip nix_integration_tests"
+    ];
+    crateOverrides = defaultCrateOverrides // {
+      crate2nix = { src, ... }: {
+        src =
+          if release
+          then src
+          else lib.cleanSourceWith {
+            filter = withoutTemplates;
+            inherit src;
+          };
+          dontFixup = !release;
+      };
+      cssparser-macros = attrs: assert builtins.trace "cssparser" true;{
+        buildInputs = stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
+      };
+    };
+  };
+  set_templates = if release then "" else "--set TEMPLATES_DIR ${./crate2nix/templates}";
+in
+symlinkJoin {
+  name = crate2nix.name;
+  paths = [ crate2nix ];
+  buildInputs = [ makeWrapper cargo ];
+  meta = {
+    description = "Nix build file generator for rust crates.";
+    longDescription = ''
+      Crate2nix generates nix files from Cargo.toml/lock files
+      so that you can build every crate individually in a nix sandbox.
+    '';
+    homepage = https://github.com/kolloch/crate2nix;
+    license = lib.licenses.asl20;
+    maintainers = [
+      {
+        github = "kolloch";
+        githubId = 339354;
+        name = "Peter Kolloch";
+      }
+      lib.maintainers.andir
+    ];
+    platforms = lib.platforms.all;
+  };
+  postBuild = ''
+    # Fallback to built dependencies for cargo and nix-prefetch-url
+    wrapProgram $out/bin/crate2nix ${set_templates}\
+    --suffix PATH ":" ${lib.makeBinPath [ cargo nix nix-prefetch-git ]}
```

Comment:
CI is crashing with `Builder called die: makeWrapper doesn't understand the arg PATH`

I see this is the same as the old `default.nix`, so I'm not sure how this passed CI before...

https://github.com/kolloch/crate2nix/pull/151/checks?check_run_id=1144007710#step:5:550